src/js/controller/settings/exportimage/GifExportController.js

```diff
@@ -174,7 +176,7 @@
         link : imageUrl,
         shortLink : this.shorten_(imageUrl, URL_MAX_LENGTH, '...')
       });
-      this.uploadStatusContainerEl.innerHTML = 'Your image is now available at : ' + linkHtml;
+      this.uploadStatusContainerEl.innerHTML = this.i18n.gifExportSettingSectionYourImageAvailableAt() + ' ' + linkHtml;
```

Comment:
Instead of string concatination, use a javascript string [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).
If you use backticks instead of double or single quotes, you can inject variables directly into the string. This makings the string easier to read and spacing harder to mess up.
```suggestion
      this.uploadStatusContainerEl.innerHTML = `${this.i18n.gifExportSettingSectionYourImageAvailableAt()} ${linkHtml}`;
```

Reply:
Good practice. I am going to start using template literals instead!